treeherder/changelog/collector.py

```diff
@@ -0,0 +1,97 @@
+""" Collector, grabs changes in various sources and put them in a DB.
+"""
+import json
+import os
+
+import github3
+from django.conf import settings
+
+from treeherder.changelog.filters import Filters
+
+MAX_ITEMS = 100
+CFG = os.path.join(os.path.dirname(__file__), "repositories.json")
+
+with open(CFG) as f:
+    CFG = json.loads(f.read())
+
+
+class GitHub:
+    def __init__(self):
+        self.token = os.environ["GITHUB_CLIENT_SECRET"]
+        self.id = os.environ["GITHUB_CLIENT_ID"]
```

Comment:
Is this id being used anywhere?

Reply:
no, good catch, we can remove it. the client_id is only used when we want to authorize for a new token, I'll remove it here